tests/scenarios/test_functionality_nimbus.py

```diff
@@ -126,3 +137,90 @@ def check_new_tab(selenium):
+
+    # install language pack
+    selenium.get("https://addons.mozilla.org/en-US/firefox/addon/acholi-ug-language-pack/")
+    selenium.find_element(By.CSS_SELECTOR, ".AMInstallButton-button").click()
+    with selenium.context(selenium.CONTEXT_CHROME):
+        WebDriverWait(selenium, 60).until(EC.element_to_be_clickable(add_to_firefox_locator))
+        time.sleep(5)  # need to sleep as the waits sometimes don't work
+        selenium.find_element(*add_to_firefox_locator).click()
+        WebDriverWait(selenium, 60).until(
+            EC.visibility_of_element_located(addon_installed_locator)
+        )
+        selenium.find_element(*addon_installed_locator).click()
+
+    selenium.get("about:preferences")
+    button = selenium.find_element(*language_button_locator)
+    button.click()
+    WebDriverWait(selenium, 60).until(EC.visibility_of_element_located(root_dialog_box_locator))
+    dialog = selenium.find_element(*root_dialog_box_locator)
+    selenium.switch_to.frame(dialog)
+
+    dialog = selenium.find_element(*browser_dialog_box_locator)
+    menu_list = selenium.find_element(*menu_list_locator)
+    menu_list.click()
+    WebDriverWait(menu_list, 60).until(EC.visibility_of_element_located(language_search_locator))
+    el = menu_list.find_element(*language_search_locator)
+    ActionChains(selenium).move_to_element(el).pause(1).click().perform()
+    ActionBuilder(selenium).clear_actions()
+    menu_list.click()
+    language_list = menu_list.find_elements(By.CSS_SELECTOR, "menuitem")
+    for item in language_list:
+        if "Acholi" in item.get_attribute("label"):
+            selenium.execute_script("arguments[0].scrollIntoView(true);", item)
+            ActionChains(selenium).move_to_element(item).pause(1).click().pause(1).perform()
+            break
+    WebDriverWait(dialog, 60).until(
+        EC.element_to_be_clickable(add_button_locator), message="Language was not added"
+    )
+    dialog.find_element(*add_button_locator).click()
+
+    time.sleep(15)  # need to use a hard wait for the langage pack to download
+
+
+@then("Firefox will be correctly localized for the installed language pack")
+def check_for_localized_firefox(selenium):
+    locales_locator = (By.CSS_SELECTOR, "#primaryBrowserLocale")
+    text_locator = (By.CSS_SELECTOR, "#browserLanguagesBox > description:nth-child(1)")
+
+    selenium.get("about:preferences")
+    locales = selenium.find_element(*locales_locator)
+    locales.click()
+    list = WebDriverWait(locales, 60).until(
+        EC.presence_of_all_elements_located((By.CSS_SELECTOR, "menuitem"))
+    )
+    for item in list:
+        if "Acholi" in item.get_attribute("label"):
+            ActionChains(selenium).move_to_element(item).pause(1).click().pause(1).perform()
+            break
+    WebDriverWait(selenium, 60).until(
+        EC.text_to_be_present_in_element(
+            text_locator,
+            "Yer leb ma kitiyo kwedgi me nyuto jami ayera, kwena, ki jami angeya ki ii Firefox.",
```

Comment:
nit: Can you add a comment above this with the English-ified version of this?
Google translate is detecting it as Hindi, and I'm not finding a lot of Acholi to English online translators that can translate this back to me, and I'm not confident in Google's answer of:
> _"This lab is based in New York City, Argentina, and is based on Firefox."_